README.rst

```diff
@@ -157,20 +159,20 @@ Configuring anycast-healthchecker
 ---------------------------------
 
 Because anycast-healthchecker is very much tied in with Bird daemon, we first
-explain the configuration you need to make in Bird. Then, we will cover the
-configuration of anycast-healthchecker daemon, and then the configuration of
-the health checks in particular and finally we'll cover the invocation of the
-program from the command line.
+explain the configuration of Bird. We will then cover the
+configuration of anycast-healthchecker (including the configuration for the 
+health checks) and finally we'll describe the options for invoking the program
+from the command line.
 
 Bird configuration
 ##################
 
 Below is an example configuration for Bird which establishes the logic described
-`How anycast-healthchecker works`_. It is the minimum configuration required by
-anycast-healthchecker to function properly.
+in `How anycast-healthchecker works`_. It is the minimum configuration required 
+to interface Bird and anycast-healthchecker.
```

Comment:
my bad here - the correct form is "interface with" see https://www.google.gr/search?num=20&hl=el&biw=1076&bih=970&q=%22interface+with%22+english+dictionary&oq=%22interface+with%22+english+dictionary&gs_l=serp.3...41601.52739.0.52960.26.25.0.0.0.0.290.2504.0j17j1.18.0....0...1c.1.64.serp..13.13.1878.zqIsGYybtFM